inc/components/namespace.php

```diff
@@ -1,16 +1,22 @@
 <?php
```

Comment:
This introduces a new file naming pattern, where functions of a namespace reside in a `namespace.php` file in the directory for that namespace. Additionally, all filters and actions get hooked from inside a `bootstrap()` function that is called from the main plugin file after the namespace files are loaded.